app/src/Molecule.re

```diff
@@ -0,0 +1,43 @@
+open Source.Project;
+open Atom;
+open Particle;
+
+module Styles = {
+  open Css;
+
+  let item =
+    style([
+      width(pct(100.0)),
+      display(`flex),
+      flex(`num(1.0)),
+      padding(px(13)),
+      borderBottom(px(1), solid, Color.lightGray()),
+      lastChild([borderBottomWidth(px(0))]),
+    ]);
+
+  let description =
+    style([
+      display(`flex),
+      flexDirection(`column),
+      justifyContent(`center),
+    ]);
+
+  let link = style([display(`flex)]);
+
+  let image =
+    style([marginRight(px(21)), width(px(64)), height(px(64))]);
+};
+
+module ProjectCard = {
+  [@react.component]
+  let make = (~project) =>
+    <li className=Styles.item key={project.address}>
```

Comment:
Should we make this a `div` or other container to not assume that it is rendered in a list?